### PR TITLE
feat(sell flow): fixed proper label to be shown on order cancel page

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx [hunk 1]
```diff
@@ -52,9 +54,10 @@ const Template: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
           >
             <FormattedMessage
               id='modals.simplebuy.cancelorder.outcome'
-              defaultMessage='Cancelling this {pair} Buy will remove your order. You can always create a new order from the menu if you cancel now.'
+              defaultMessage='Cancelling this {pair} {orderType} will remove your order. You can always create a new order from the menu if you cancel now.'
               values={{
-                pair: props.order.pair
+                pair: props.order.pair,
+                orderType: orderType === 'BUY' ? 'Buy' : 'Sell'
```
Comment (dsisto-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx:60]: Does this work on, say, spanish?

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx:60]: this is valid point, we have such hardcoded text on several pages, I'll change this to be fully dynamic and we will see for other points

Comment (dsisto-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx:60]: I don't know. You tell me! If this is common patter in the repo, let's go with it!

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx:60]: Heh it's follow up of this part https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/template.success.tsx#L325

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx:60]: @dsisto-bc let's go like this for now, tnx!

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx:60]: good call out @dsisto-bc. we have bad patterns like this all over. need to address in the future

**Review by schnogz — APPROVED** [review]

**Pull request merged**